content/docs/components-and-props.md

```diff
@@ -30,7 +30,7 @@ function Welcome(props) {
 }
 ```
 
-This function is a valid React component because it accepts a single "props" object argument with data and returns a React element. We call such components "functional" because they are literally JavaScript functions.
+This function is a valid React component because it accepts a single "props" (which stands for properties) object argument with data and returns a React element. We call such components "functional" because they are literally JavaScript functions.
```

Comment:
Disclaimer: My knowledge of grammar is mediocre.

If the meaning of "props" is unclear then I like the idea of explaining it to people. This sentence seems to read kind of awkwardly now though, given the placement of parens.

I wonder if this might be clearer?

> This function is a valid React component because it accepts a single "props" parameter (short for "properties") with data and returns a React element.

Reply:
Whoops. Looks like this was merged while I was typing 😅